lib/upgrade_analyzer/analyzer.rb

```diff
@@ -146,7 +146,7 @@ def report_invalid_comparison(github, errors)
     end
 
     def log(msg)
-      puts msg
+      puts "UpgradeAnalyzer: #{msg}"
```

Comment:
What about using a logger? So we can manage more options: decide where to output the logs, level of detail, etc.

Reply:
Done!